docs/update-strategies.md

```diff
@@ -1,6 +1,7 @@
 ## This doc is still under development
 
-* `all` strategy uses `version` passed to options as a cache tag. When version changes, old version cache is removed and new files are downloaded. Of course if files has same name were not changed (304 status returned), then probably browser won't download them again and will just update cache.
-* `changed` strategy is more advanced than `all` or `hash`. To work properly it requires output assets to have unique names between compilations, e.g. _file's unique hash_ in its name. **With this strategy enabled, `index.html` file (or other files without dynamic name) should be placed in `main` section of the cache, otherwise they won't be revalidated**.
+* **`all`** strategy uses `version` passed to `options` as a cache tag. When version changes, old version cache is removed and new files are downloaded. Of course if files with the same name were not changed (HTTP status 304), the browser probably won't download them again and will just update cache.
```

Comment:
I try to use "code" style only for things from the code, e.g. JS value or option name. So not sure about `options`.

Reply:
You know what? Ignore me about being dull and that `options` thing :-) Maybe just make it a link to the options doc?

Reply:
I added a link.. It's still formatted as code 😆 let me know if you want it changed